src/olympia/discovery/data.py

```diff
@@ -19,8 +19,8 @@ def __init__(self, *args, **kwargs):
                   '{end_sub_heading}'),
         description=string_concat(
             '<blockquote>',
-            _(u'Remove blaring ads and make the Web run smoother! Adblock '
-              u'Plus can also mitigate tracking mechanisms and malware.'),
+            _(u'Remove blaring ads and make the Web run smoother with ')
```

Comment:
Was fixed in https://github.com/mozilla/addons-server/commit/0c2b633c592a882a51698949f2262773d1462b5b